docs/_config.yaml

```diff
@@ -1,3 +1,45 @@
-# _config.yml
-
-title: "Jekyll Actions Demo"
+theme: minimal-mistakes-jekyll
+minimal_mistakes_skin: "air"
+locale: "en-US"
+title: "The Kubernetes NMState project"
+description: A Declarative API for Kubernetes Node Network Management
```

Comment:
Since this is just a description and not a title, it should use normal capitalization:

A declarative API for Kubernetes node network management